assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -98,8 +98,7 @@
 
 .productView-thumbnail {
     @include grid-column(3);
-    padding-left: spacing("quarter");
-    padding-right: spacing("quarter");
+    padding: spacing("quarter");
```

Comment:
just clarification: is it intended to add spacing for top and bottom as well?

Reply:
yes, it resolves cutted outline of nested link